use prefix to distinguish dimensions from different span kind (#16003)
open-telemetry/opentelemetry-collector-contrib: processor/servicegraphprocessor/README.md

Possible values for `connection_type`: unset, `messaging_system`, or `database`.

> Additional labels can be included using the `dimensions` configuration option. Those labels will have a prefix to mark where are from.

Reviewer: ```suggestion
Additional labels can be included using the `dimensions` configuration option. Those labels will have a prefix to mark where they originate (client or server span kinds).
```

> The `client_` prefix regards the dimensions come from a span with `SPAN_KIND_CLIENT`, and the `server_` prefix is regard as the

Reviewer: ```suggestion
The `client_` prefix relates to the dimensions coming from spans with `SPAN_KIND_CLIENT`, and the `server_` prefix relates to the
```

> dimensions come from a span with `SPAN_KIND_SERVER`.

Reviewer: ```suggestion
dimensions coming from spans with `SPAN_KIND_SERVER`.
```
